db/R/query.format.vars.R

```diff
@@ -112,6 +112,22 @@ query.format.vars <- function(bety,input.id=NA,format.id=NA,var.ids=NA){
                    lat = site.lat,
                    lon = site.lon
     )
+    
+    # Check that all bety units are convertible. If not, throw a warning. 
+    for(i in 1:length(format$vars$bety_units)){
+      
+      if( format$vars$storage_type[i] != ""){ #units with storage type are a special case
+        
+        # This would be a good place to put a test for valid sotrage types. Currently not implemented. 
+        
+      }else if(udunits2::ud.are.convertible(format$vars$input_units[i], format$vars$pecan_units[i]) == FALSE){ 
+        
+        if(misc.are.convertible(format$vars$input_units[i], format$vars$pecan_units[i]) == FALSE){
```

Comment:
@bcow @mccabete not super important as this gives a warning not  an error but I think there needs to be another condition here, checking for same units..

Because above input_units are populated with bety_units, and now there are cases where input_units and pecan_units the same but are not convertible according to this check therefore meets this condition.

E.g. (In Ameriflux format) for variable CO2 both inputs_units and pecan_units are "ppm (= umol mol-1)" and this check gives:

   ```
Units not convertible for CO2 with units of ppm (= umol mol-1) .  Please 
   make sure the varible has units that can be converted to ppm (= umol 
   mol-1) 
```